source/_integrations/plugwise.markdown

```diff
@@ -25,6 +25,7 @@ climate:
   - platform: plugwise
     password: YOUR_SHORT_IP 
     host: YOUR_SMILE_LOCAL_IP
+    legacy_anna: true # required for a legacy Anna (firmware 1.8.x), not needed for a more recent Anna (firmware 3.1.x)
```

Comment:
We want only required config variables in the first example, you can put the explenation in the description for `legacy_anna`

Reply:
Rebased and changes made.